### PR TITLE
Execute and Get File API documentation

### PR DESCRIPTION
**Pull request by kevinlog**
Adds API documentation for the `execute` and `get-file` response action

### CODE REVIEW

File: docs/management/api/get-file-api.asciidoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+===== Example response
+
+[source,json]
+--------------------------------------------------
+{
+  "data": {
+    "id": "27ba1b42-7cc6-4e53-86ce-675c876092b2",
+      "agents": [
+          "2b292af2-5eb3-4d74-8a80-1ec6b56f66ac"
```
Comment (natasha-moore-elastic) [inline docs/management/api/get-file-api.asciidoc:65]: Should this value match the `endpoint_id` value from the request example?
Same question for `hosts` and `agentState` values.

File: docs/management/api/get-file-api.asciidoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+A JSON object with the details of the response action created.
+
+===== Example response
+
+[source,json]
+--------------------------------------------------
+{
+  "data": {
+    "id": "27ba1b42-7cc6-4e53-86ce-675c876092b2",
+      "agents": [
```
Comment (natasha-moore-elastic) [inline docs/management/api/get-file-api.asciidoc:64]: Should `agents` and `hosts` be nested under `id` or should they have the same indentation as `id` (as in the Execute a command example?)

File: docs/management/api/execute-api.asciidoc [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+A JSON object with the details of the response action created.
+
+===== Example response
+
+[source,json]
+--------------------------------------------------
+{
+  "data": {
+    "id": "9f934028-2300-4927-b531-b26376793dc4",
```
Comment (natasha-moore-elastic) [inline docs/management/api/execute-api.asciidoc:65]: Seems like the indentation across the example is a bit inconsistent (i.e. a mixture of 2-space and 4-space indents).

**Review by natasha-moore-elastic — COMMENTED** [review]
Thanks so much for raising this PR @kevinlog! Just left a few questions and comments.

File: docs/management/api/get-file-api.asciidoc (hunk 2 above)
Comment (kevinlog) [reply docs/management/api/get-file-api.asciidoc:64]: good catch, they should be the same indentation as `id`, will push up the change

File: docs/management/api/execute-api.asciidoc (hunk 3 above)
Comment (kevinlog) [reply docs/management/api/execute-api.asciidoc:65]: I'm correcting the indentation in the next push

File: docs/management/api/get-file-api.asciidoc [hunk 4]
```diff
@@ -0,0 +1,92 @@
+[[get-file-api]]
+=== Get a file from a host
+
+Get a file from a host running {elastic-defend}.
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/management/api/get-file-api.asciidoc:4]: +1

**Comment by natasha-moore-elastic** [comment]
* Resolves https://github.com/elastic/security-docs/issues/3421.

**Review by natasha-moore-elastic — APPROVED** [review]

**Pull request merged**